src/environment/src/env/GazeboEnv.py

```diff
@@ -82,14 +91,11 @@ def render(self, mode='human'):
 
     def _get_observation(self):
         """
-
         :return: State
         """
-        left_image = self._get_image_data_from_topic(self.LEFT_CAMERA_TOPIC)
-        center_image = self._get_image_data_from_topic(self.CENTER_CAMERA_TOPIC)
-        right_image = self._get_image_data_from_topic(self.RIGHT_CAMERA_TOPIC)
 
-        state = State(left_image, center_image, right_image)
+        center_image = self.center_image.image
+        state = State(None, center_image, None)
```

Comment:
Maybe we could throw away State class and we will use 1 picture everywhere? We will probably have only one camera and we dont use other images in other places

Reply:
I removed state form this class.